fanotify.go

```diff
@@ -47,7 +47,7 @@ func NewFanotifyWatcher() (*FanotifyWatcher, error) {
 func (fw *FanotifyWatcher) Add(path string) error {
 	err := unix.FanotifyMark(
 		fw.fd,
-		unix.FAN_MARK_ADD|unix.FAN_MARK_MOUNT,
+		unix.FAN_MARK_ADD|unix.FAN_MARK_MOUNT|unix.FAN_MOVED_TO,
 		unix.FAN_CLOSE_WRITE,
```

Comment:
I guess we have to add this here, i.e. unix.FAN_CLOSE_WRITE | unix.FAN_MOVED_TO.

See http://man7.org/linux/man-pages/man7/fanotify.7.html:
       The bits that may appear in mask are as follows:
       ...
       FAN_MOVED_TO
              A file or directory has been moved to a watched parent direc‐
              tory.
       ...
       FAN_CLOSE_WRITE
              A file that was opened for writing (O_WRONLY or O_RDWR) was
              closed.